Docgen/business/plugins/org.polarsys.kitalpha.doc.gen.business.core/templates/pattern._0T5DwF4yEeGfnvXFIF-hXQ/method._0T5DxV4yEeGfnvXFIF-hXQ.pt

```diff
@@ -50,7 +50,7 @@ int i = 0;
 for(Entry<String, IndexItem> entry : indexItems.entrySet()) {
 		i++;
 		%>
-		conceptsList[<%=listCount%>][<%=count%>][0] = "<%=EscapeChars.removeNewline(EscapeChars.forHTML(entry.getValue().getConceptName().replace("\"", "\\\"")))%>";
```

Comment:
What are the impacts of this modification? can all special characters be added in the generated html page?

Reply:
Yes in fact they can.

The modification integrated in commit 57c624f have been tested with the following:

![image](https://user-images.githubusercontent.com/1576016/100086707-1041b300-2e4e-11eb-9a33-6ad8d5a7433a.png)

Reply:
And it works !